ui/src/message_popup/playtest/hmtca_group_review.jsx

```diff
@@ -1,4 +1,4 @@
-/* @flow weak */
+// TODO(kr) flow typing disabled because of error in refreshTimer typing
```

Comment:
@milsim11 This is a hack workaround for now since we have 10 minutes :)  This removes the annotation at the top that tells Flow to type check this file.